src/main/kotlin/nl/info/client/brp/exception/BrpInvalidPurposeException.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.brp.exception
+
+class BrpInvalidPurposeException(message: String, cause: Throwable?) : RuntimeException(message, cause) {
```

Comment:
Minor thing, but I don't see that we use this constructor passing a `cause`, so I see this as prematurely anticipating that we will use it at some point. There is no need to do that, it'll be easy enough to add if we do need it in the future.